engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties

```diff
@@ -1,27 +1,53 @@
+org.hibernate.validator.constraints.CodePointLength.message         = d\u00e9lka mus\u00ed le\u017eet v rozsahu {min} a\u017e {max}
+org.hibernate.validator.constraints.LuhnCheck.message               = Kontroln\u00ed \u010d\u00edslice pro ${validatedValue} je neplatn\u00e1, kontroln\u00ed sou\u010det Luhn Modulo 10 se nezda\u0159il
+org.hibernate.validator.constraints.Mod10Check.message              = Kontroln\u00ed \u010d\u00edslice pro ${validatedValue} je neplatn\u00e1, kontroln\u00ed sou\u010det Modulo 10 se nezda\u0159il
+org.hibernate.validator.constraints.Mod11Check.message              = Kontroln\u00ed \u010d\u00edslice pro ${validatedValue} je neplatn\u00e1, kontroln\u00ed sou\u010det Modulo 11 se nezda\u0159il
+org.hibernate.validator.constraints.ModCheck.message                = Kontroln\u00ed \u010d\u00edslice pro ${validatedValue} je neplatn\u00e1, kontroln\u00ed sou\u010det ${modType} se nezda\u0159il
+org.hibernate.validator.constraints.NotBlank.message                = nesm\u00ed b\u00fdt pr\u00e1zdn\u00e1
+org.hibernate.validator.constraints.NotEmpty.message                = nesm\u00ed b\u00fdt pr\u00e1zdn\u00e1
+org.hibernate.validator.constraints.ParametersScriptAssert.message  = v\u00fdraz skriptu "{script}" se nevyhodnotil na true
+org.hibernate.validator.constraints.Range.message                   = mus\u00ed le\u017eet v rozsahu {min} a\u017e {max}
+org.hibernate.validator.constraints.SafeHtml.message                = je mo\u017en\u00e9, \u017ee obsah html nen\u00ed bezpe\u010dn\u00fd
+org.hibernate.validator.constraints.ScriptAssert.message            = v\u00fdraz skriptu "{script}" se nevyhodnotil na true
+org.hibernate.validator.constraints.UniqueElements.message          = mus\u00ed obsahovat pouze jedine\u010dn\u00e9 prvky
+org.hibernate.validator.constraints.URL.message                     = mus\u00ed b\u00fdt platnou adresou URL
+
+org.hibernate.validator.constraints.br.CNPJ.message                 = neplatn\u00e9 brazilsk\u00e9 registra\u010dn\u00ed \u010d\u00edslo pl\u00e1tce korpor\u00e1tn\u00ed dan\u011b (CNPJ)
+org.hibernate.validator.constraints.br.CPF.message                  = neplatn\u00e9 brazilsk\u00e9 registra\u010dn\u00ed \u010d\u00edslo individu\u00e1ln\u00edho pl\u00e1tce dan\u011b (CPF)
+org.hibernate.validator.constraints.br.TituloEleitoral.message      = neplatn\u00e9 brazilsk\u00e9 \u010d\u00edslo karty Voter ID
+
+org.hibernate.validator.constraints.pl.REGON.message                = Neplatn\u00e9 polsk\u00e9 identifika\u010dn\u00ed \u010d\u00edslo pl\u00e1tn\u011b dan\u011b (REGON)
+org.hibernate.validator.constraints.pl.NIP.message                  = Neplatn\u00e9 da\u0148ov\u00e9 identifika\u010dn\u00ed \u010d\u00edslo (NIP)
+org.hibernate.validator.constraints.pl.PESEL.message                = Neplatn\u00e9 polsk\u00e9 n\u00e1rodn\u00ed identifika\u010dn\u00ed \u010d\u00edslo (PESEL)
```

Comment:
The case is not very consistent here. In the original file, we decided to use capitals for all the parts of the number as it's the translation of an acronym. Here we only have the first element capitalized and not the others. Either we decide to stick to what we did in the English version or we are consistent with lowercase if it's better in the language.